httpcache.go

```diff
@@ -98,6 +127,10 @@ type Transport struct {
 	Cache     Cache
 	// If true, responses returned from the cache will be given an extra header, X-From-Cache
 	MarkCachedResponses bool
+	// Mapping of original request => cloned
+	modReq map[*http.Request]*http.Request
+	// guards clonedReqs
+	mu sync.RWMutex
```

Comment:
Can mu move above modReq, and have the comment use the same varname (s/clonedReqs/modReq)?